packages/core/core-flows/src/order/workflows/return/confirm-receive-return-request.ts

```diff
@@ -40,13 +48,95 @@ const validationStep = createStep(
   }
 )
 
+function prepareInventoryUpdate({ orderReturn, returnedQuantityMap }) {
```

Comment:
nit: Can I get you to add a comment describing the logic in this function? I think it's a bit hard to follow, so some context might help for the next reader.

Reply:
I added a couple of comments along the function.
I hope that helps a bit

Reply:
This is great, thanks!